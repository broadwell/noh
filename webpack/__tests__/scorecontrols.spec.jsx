import React from "react";
import { Provider } from "react-redux";
import { shallow, mount } from "enzyme";
import configureMockStore from "redux-mock-store";
import ScoreControls, {
  Unwrapped as UnwrappedScoreControls
} from "../components/ScoreControls";

describe("<ScoreControls>", () => {
  let wrapper;
  let store;

  beforeEach(() => {
    const mockStore = configureMockStore();
    const initialState = {};
    store = mockStore(initialState);
    wrapper = mount(
      <Provider store={store}>
        <ScoreControls updateScoreToggles={jest.fn()} />
      </Provider>
    );
  });

  it("renders as expected by default", () => {
    const component = shallow(<UnwrappedScoreControls />);
    expect(component).toMatchSnapshot();
  });

  it("renders as expected by default with a store", () => {
    expect(wrapper).toMatchSnapshot();
  });

  fit("handles checkbox changes", () => {
    const action = { type: "SET_SCORE_TOGGLES", payload: {} };
    wrapper.find("input").forEach(input => {
      input.simulate("change");
      expect(store.getActions()[0].type).toEqual(action.type);
    });
  });
});
