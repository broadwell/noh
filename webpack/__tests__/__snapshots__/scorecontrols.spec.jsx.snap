// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ScoreControls> renders as expected by default 1`] = `
<div
  className="score-controls"
>
  <ul
    className="channel-toggles"
  >
    <li>
      <input
        checked={true}
        id="scoreBeat"
        onChange={[Function]}
        onKeyPress={null}
        type="checkbox"
      />
      <label
        htmlFor="scoreBeat"
      >
        <svg
          height="12"
          viewBox="0 0 12 12"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="#9aa0a6"
            fillRule="evenodd"
          >
            <path
              d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
              fillRule="nonzero"
            />
            <path
              d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
              transform="rotate(-45 6.152 5.5)"
            />
          </g>
        </svg>
        Beat
      </label>
    </li>
    <li>
      <input
        checked={true}
        id="scoreText"
        onChange={[Function]}
        onKeyPress={null}
        type="checkbox"
      />
      <label
        htmlFor="scoreText"
      >
        <svg
          height="12"
          viewBox="0 0 12 12"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="#9aa0a6"
            fillRule="evenodd"
          >
            <path
              d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
              fillRule="nonzero"
            />
            <path
              d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
              transform="rotate(-45 6.152 5.5)"
            />
          </g>
        </svg>
        Text
      </label>
    </li>
    <li>
      <input
        checked={true}
        id="scorePercussion"
        onChange={[Function]}
        onKeyPress={null}
        type="checkbox"
      />
      <label
        htmlFor="scorePercussion"
      >
        <svg
          height="12"
          viewBox="0 0 12 12"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="#9aa0a6"
            fillRule="evenodd"
          >
            <path
              d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
              fillRule="nonzero"
            />
            <path
              d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
              transform="rotate(-45 6.152 5.5)"
            />
          </g>
        </svg>
        Percussion
      </label>
    </li>
    <li>
      <input
        checked={true}
        id="scoreNohkan"
        onChange={[Function]}
        onKeyPress={null}
        type="checkbox"
      />
      <label
        htmlFor="scoreNohkan"
      >
        <svg
          height="12"
          viewBox="0 0 12 12"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="#9aa0a6"
            fillRule="evenodd"
          >
            <path
              d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
              fillRule="nonzero"
            />
            <path
              d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
              transform="rotate(-45 6.152 5.5)"
            />
          </g>
        </svg>
        Nohkan
      </label>
    </li>
    <li>
      <input
        checked={true}
        id="scoreDance"
        onChange={[Function]}
        onKeyPress={null}
        type="checkbox"
      />
      <label
        htmlFor="scoreDance"
      >
        <svg
          height="12"
          viewBox="0 0 12 12"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="#9aa0a6"
            fillRule="evenodd"
          >
            <path
              d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
              fillRule="nonzero"
            />
            <path
              d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
              transform="rotate(-45 6.152 5.5)"
            />
          </g>
        </svg>
        Dance
      </label>
    </li>
  </ul>
  <ul
    className="measure-toggles"
  >
    <li>
      <input
        checked={true}
        id="scorePrevSentence"
        onChange={[Function]}
        onKeyPress={null}
        type="checkbox"
      />
      <label
        htmlFor="scorePrevSentence"
      >
        <svg
          height="12"
          viewBox="0 0 12 12"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="#9aa0a6"
            fillRule="evenodd"
          >
            <path
              d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
              fillRule="nonzero"
            />
            <path
              d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
              transform="rotate(-45 6.152 5.5)"
            />
          </g>
        </svg>
        Previous Sentence
      </label>
    </li>
    <li>
      <input
        checked={true}
        id="scoreNextSentence"
        onChange={[Function]}
        onKeyPress={null}
        type="checkbox"
      />
      <label
        htmlFor="scoreNextSentence"
      >
        <svg
          height="12"
          viewBox="0 0 12 12"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="#9aa0a6"
            fillRule="evenodd"
          >
            <path
              d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
              fillRule="nonzero"
            />
            <path
              d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
              transform="rotate(-45 6.152 5.5)"
            />
          </g>
        </svg>
        Next Sentence
      </label>
    </li>
  </ul>
</div>
`;

exports[`<ScoreControls> renders as expected by default with a store 1`] = `
<Provider
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Connect(ScoreControls)
    updateScoreToggles={[Function]}
  >
    <ScoreControls
      isBeatOn={true}
      isDanceOn={true}
      isNextSentenceOn={true}
      isNohkanOn={true}
      isPercussionOn={true}
      isPrevSentenceOn={true}
      isTextOn={true}
      updateScoreToggles={[Function]}
    >
      <div
        className="score-controls"
      >
        <ul
          className="channel-toggles"
        >
          <li>
            <input
              checked={true}
              id="scoreBeat"
              onChange={[Function]}
              onKeyPress={null}
              type="checkbox"
            />
            <label
              htmlFor="scoreBeat"
            >
              <svg
                height="12"
                viewBox="0 0 12 12"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#9aa0a6"
                  fillRule="evenodd"
                >
                  <path
                    d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
                    fillRule="nonzero"
                  />
                  <path
                    d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
                    transform="rotate(-45 6.152 5.5)"
                  />
                </g>
              </svg>
              Beat
            </label>
          </li>
          <li>
            <input
              checked={true}
              id="scoreText"
              onChange={[Function]}
              onKeyPress={null}
              type="checkbox"
            />
            <label
              htmlFor="scoreText"
            >
              <svg
                height="12"
                viewBox="0 0 12 12"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#9aa0a6"
                  fillRule="evenodd"
                >
                  <path
                    d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
                    fillRule="nonzero"
                  />
                  <path
                    d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
                    transform="rotate(-45 6.152 5.5)"
                  />
                </g>
              </svg>
              Text
            </label>
          </li>
          <li>
            <input
              checked={true}
              id="scorePercussion"
              onChange={[Function]}
              onKeyPress={null}
              type="checkbox"
            />
            <label
              htmlFor="scorePercussion"
            >
              <svg
                height="12"
                viewBox="0 0 12 12"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#9aa0a6"
                  fillRule="evenodd"
                >
                  <path
                    d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
                    fillRule="nonzero"
                  />
                  <path
                    d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
                    transform="rotate(-45 6.152 5.5)"
                  />
                </g>
              </svg>
              Percussion
            </label>
          </li>
          <li>
            <input
              checked={true}
              id="scoreNohkan"
              onChange={[Function]}
              onKeyPress={null}
              type="checkbox"
            />
            <label
              htmlFor="scoreNohkan"
            >
              <svg
                height="12"
                viewBox="0 0 12 12"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#9aa0a6"
                  fillRule="evenodd"
                >
                  <path
                    d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
                    fillRule="nonzero"
                  />
                  <path
                    d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
                    transform="rotate(-45 6.152 5.5)"
                  />
                </g>
              </svg>
              Nohkan
            </label>
          </li>
          <li>
            <input
              checked={true}
              id="scoreDance"
              onChange={[Function]}
              onKeyPress={null}
              type="checkbox"
            />
            <label
              htmlFor="scoreDance"
            >
              <svg
                height="12"
                viewBox="0 0 12 12"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#9aa0a6"
                  fillRule="evenodd"
                >
                  <path
                    d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
                    fillRule="nonzero"
                  />
                  <path
                    d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
                    transform="rotate(-45 6.152 5.5)"
                  />
                </g>
              </svg>
              Dance
            </label>
          </li>
        </ul>
        <ul
          className="measure-toggles"
        >
          <li>
            <input
              checked={true}
              id="scorePrevSentence"
              onChange={[Function]}
              onKeyPress={null}
              type="checkbox"
            />
            <label
              htmlFor="scorePrevSentence"
            >
              <svg
                height="12"
                viewBox="0 0 12 12"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#9aa0a6"
                  fillRule="evenodd"
                >
                  <path
                    d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
                    fillRule="nonzero"
                  />
                  <path
                    d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
                    transform="rotate(-45 6.152 5.5)"
                  />
                </g>
              </svg>
              Previous Sentence
            </label>
          </li>
          <li>
            <input
              checked={true}
              id="scoreNextSentence"
              onChange={[Function]}
              onKeyPress={null}
              type="checkbox"
            />
            <label
              htmlFor="scoreNextSentence"
            >
              <svg
                height="12"
                viewBox="0 0 12 12"
                width="12"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#9aa0a6"
                  fillRule="evenodd"
                >
                  <path
                    d="M6,11.5 C2.96243388,11.5 0.5,9.03756612 0.5,6 C0.5,2.96243388 2.96243388,0.5 6,0.5 C9.03756612,0.5 11.5,2.96243388 11.5,6 C11.5,9.03756612 9.03756612,11.5 6,11.5 Z M6,10.5 C8.48528137,10.5 10.5,8.48528137 10.5,6 C10.5,3.51471863 8.48528137,1.5 6,1.5 C3.51471863,1.5 1.5,3.51471863 1.5,6 C1.5,8.48528137 3.51471863,10.5 6,10.5 Z"
                    fillRule="nonzero"
                  />
                  <path
                    d="M4.90165043,6 L7.90165043,6 C8.1777928,6 8.40165043,6.22385763 8.40165043,6.5 C8.40165043,6.77614237 8.1777928,7 7.90165043,7 L4.40165043,7 C4.12550805,7 3.90165043,6.77614237 3.90165043,6.5 L3.90165043,4.5 C3.90165043,4.22385763 4.12550805,4 4.40165043,4 C4.6777928,4 4.90165043,4.22385763 4.90165043,4.5 L4.90165043,6 Z"
                    transform="rotate(-45 6.152 5.5)"
                  />
                </g>
              </svg>
              Next Sentence
            </label>
          </li>
        </ul>
      </div>
    </ScoreControls>
  </Connect(ScoreControls)>
</Provider>
`;
